js/geo/transform.js

```diff
@@ -260,6 +260,16 @@ Transform.prototype = {
     },
 
     /**
+     * Like `locationPoint` but returns a raw 4d vector containing info about the transform
+     * @param {LngLat} lnglat location
+     * @returns {Array} The 4d vector pertaining to this point's transform
+     * @private
+     */
+    locationPoint3d: function(lnglat) {
+        return this.coordinatePoint3d(this.locationCoordinate(lnglat));
+    },
```

Comment:
What do you think about extending `Point` such that it includes a 3rd dimension rather than adding new methods to create a 3d point?